pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -84,11 +88,13 @@ var _ ActualStateOfWorld = &actualStateOfWorld{}
 
 // PluginInfo holds information of a plugin
 type PluginInfo struct {
-	SocketPath string
-	Timestamp  time.Time
-	UUID       types.UID
-	Handler    PluginHandler
-	Name       string
+	SocketPath        string
+	Timestamp         time.Time
+	UUID              types.UID
+	Handler           PluginHandler
+	Name              string
+	Endpoint          string
```

Comment:
Can you document the difference between `SocketPath` and `Endpoint`?

I think I can guess it, but it helps to be more explicit.

Reply:
done